cpp/include/cudf/functions.h

```diff
@@ -796,6 +796,42 @@ gdf_error gdf_min(gdf_column *col, void *dev_result, gdf_size_type dev_result_si
  * --------------------------------------------------------------------------*/
 gdf_error gdf_max(gdf_column *col, void *dev_result, gdf_size_type dev_result_size);
 
+/* --------------------------------------------------------------------------*
+ * @brief  Computes the rolling window function of the values in a column.
+ *
+ * This matches Pandas' API for DataFrame.rolling with a couple notable
```

Comment:
I think it's a good idea to have a self-contained description of what a function does rather than relying on someone to know what the Pandas `DataFrame.rolling` is. 

It's fine to keep the documentation of what the deviations from the Pandas API are.